tests/mock_classes/redis.py

```diff
@@ -16,6 +16,8 @@ def test_xyz():
 
     def __init__(self, **kwargs):
         """Initialize simple dict used instead of Redis storage."""
+        self.kwargs = kwargs
+        print(kwargs)
```

Comment:
Is this print just for debugging purposes?

Reply:
yes. by default we suppress prints, so it is harmles

Reply:
Acknowledged. Thank you 👍🏼

Reply:
see @onmete ? it's not just me who likes to print stuff in tests :)

Reply:
![image](https://github.com/openshift/lightspeed-service/assets/37311499/ca91be65-52c2-4efb-a512-ced7b2995ad9)